regulations/static/regulations/js/source/views/comment/comment-index-view.js

```diff
@@ -8,10 +8,11 @@ Backbone.$ = $;
 var MainEvents = require('../../events/main-events');
 var DrawerEvents = require('../../events/drawer-events');
 var CommentEvents = require('../../events/comment-events');
+var comments = require('../../collections/comment-collection');
```

Comment:
It's internally consistent. We defined a `CommentCollection` class in `comment-collection.js` but export a singleton called `comments` (lower-cased because it's an instance). I'm importing that instance using the same name throughout.